lib/Optimizer/Dialect/Quake/QuakeOps.cpp

```diff
@@ -7,6 +7,7 @@
  ******************************************************************************/
 
 #include "cudaq/Optimizer/Dialect/Quake/QuakeOps.h"
+#include "cudaq/Optimizer/Dialect/CC/CCOps.h"
```

Comment:
From the changes, I don't see why this is needed. It's a bit odd to include dialect B into the implementation specifics for dialect A.

Reply:
Leftover code. Removed in 98e4ceb295f76eb76396c571a6b44e9a91b55562